includes/shortcodes/class-wc-shortcode-products.php

```diff
@@ -196,7 +196,7 @@ protected function parse_query_args() {
 			$this->attributes['limit'] = $this->attributes['columns'] * $this->attributes['rows'];
 		}
 
-		$ordering_args         = WC()->query->get_catalog_ordering_args( $query_args['orderby'], $query_args['order'] );
+		$ordering_args         = ( $query_args['orderby'] ) ? WC()->query->get_catalog_ordering_args( $query_args['orderby'], $query_args['order'] ) : WC()->query->get_catalog_ordering_args();
```

Comment:
Let's keep this line as it was and rather just change the default $this->attributes values for order and orderby.

Have a look at the `parse_attributes` method in this class and remove the default values for order and orderby to be blanks ie. `'order'          => 'ASC'` and `'orderby'        => 'title',` since WC()->query->get_catalog_ordering_args checks for empty strings before loading the defaults.